tests/security/tpm2/tpm2_env_setup.pm

```diff
@@ -24,6 +24,7 @@ use base 'opensusebasetest';
 use testapi;
 use utils 'zypper_call';
 use power_action_utils "power_action";
```

Comment:
Looks like we need to use the same sign (" ") or (' ') for calling a api function. 
We could need to make them as the same style in one pm file.

Reply:
Got it! will do